src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1990,6 +1990,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         if (!_IsClosing())
         {
             _closing = true;
+            if (_automationPeer)
+            {
+                auto autoPeerImpl{ winrt::get_self<implementation::TermControlAutomationPeer>(_automationPeer) };
+                autoPeerImpl->IsClosing(true);
+            }

```

Comment:
I don't like this too too much, but I don't know if there's a better way to do this. `IsClosing()` is a public function off the impl type, so that helps ensure it isn't abused.

I'm mildly annoyed that we only use it _once_, but `_closing` is only set in one spot so I'll take what I can get haha.